### DISCUSSION TITLE
Отправить на почту файлы из таблицы.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#529] [Как это сделать?] [answered]
Есть задача отправить на почту файлы из таблицы.
example3: emailSend(to: "test@example.ru"; title: "Отправка письма"; body: "Тестовое письмо"; files: #file_field)
Как сделать отправку файлов типа selectlist ?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Те у вас есть несколько строк, в каждой поле файл, там один или несколько файлов и вам нужно отправить из нескольких строк одновременно?

Выглядит так, что вам массив списков, полученный при selectList надо склеить через listTrain (что бы получить список одной размерности = на одном уровне) и все.

**Reply by vldekov** [reply]
Получилось. Спасибо.
